dubbo-config/dubbo-config-api/src/main/java/org/apache/dubbo/config/bootstrap/DubboBootstrap.java

```diff
@@ -1141,6 +1163,16 @@ private void unreferServices() {
         cache.destroyAll();
     }
 
+    private ExecutorService getExportReferExecutor() {
```

Comment:
Please consider moving this executor to ExecutorRepository

Reply:
done